devops/llvm-anybuild.yml

```diff
@@ -0,0 +1,361 @@
+    default: 24
+
+resources:
+  repositories:
+  - repository: self
+  - repository: llvm
+    type: github
+    endpoint: LLVMServiceConnection
+    name: llvm/llvm-project
+
+schedules:
+- cron: "0 3 * * *" # this is 3am UTC
+  displayName: Nightly AnyBuild LLVM Build
+  always: true
+  branches:
+    include:
+    - master
+
+variables:
+- name: AbEnvironmentName
+  value: LLVMVna
+- name: AbClusterId
+  value: 0310c9eb-18d0-4a58-bc85-bdc02a4d764a
+- name: AnyBuildEnvironmentUri
+  value: https://northeurope.anybuild-test.microsoft.com/api/clusters/$(AbClusterId)/agents
+- name: AnyBuildSource
+  value: https://anybuild$(AbEnvironmentName)neurope.blob.core.windows.net/clientreleases
+- name: AnyBuildRing
+  value: Dogfood
+- name: VeronaRepoGitUri
+  value: https://github.com/microsoft/verona
+- name: LlvmRepoGitUri
+  value: https://github.com/llvm/llvm-project
+# GOOD_HASH may be set in the pipeline in which case it overrides LLVM_COMMIT 
+# - name: GOOD_HASH
+#   value: ''
+
+jobs:
+### Configure: Determine commits to build
+###   - outputs VERONA_COMMIT and LLVM_COMMIT that correspond to the latest checked in change in verona@master
+- job: Configure
+  displayName: Configure LLVM/Verona commits to build
+  pool:
+    vmImage: ubuntu-18.04
+  timeoutInMinutes: 5
+  steps:
+  - checkout: self
+
+  - bash: |
+      set -euo pipefail
+
+      VERONA_COMMIT=$(git rev-parse HEAD)
+      if [[ -z "$(GOOD_HASH)" ]]; then
+        LLVM_COMMIT=$(git submodule | grep llvm-project | cut -f1 -d\ | sed 's/^.//g')
+      else
+        LLVM_COMMIT="$(GOOD_HASH)"
+      fi
+
+      echo "Setting the following output variables:"
+      echo "  VERONA_COMMIT=$VERONA_COMMIT"
+      echo "  LLVM_COMMIT=$LLVM_COMMIT"
+      echo "##vso[task.setvariable variable=VERONA_COMMIT;isOutput=true;]$VERONA_COMMIT"
+      echo "##vso[task.setvariable variable=LLVM_COMMIT;isOutput=true;]$LLVM_COMMIT"
+    displayName: SetCommits
+    name: SetCommits
+
+############################################# Windows Builds
+
+### Build LLVM with AnyBuild first
+- job: LLVM_Windows
+  displayName: LLVM Windows
+  condition: and(succeeded(), ${{ eq(parameters.BuildOnWindows, true) }})
+  dependsOn: [ Configure ]
+  pool:
+    vmImage: windows-2019
+  timeoutInMinutes: 120
+  strategy:
+    matrix:
+      Release:
+        BuildType: Release
+        BuildName: release
+  variables:
+  - name: AbLogsRootDir
+    value: $(Build.StagingDirectory)\ab_logs
+  - name: LlvmSourceDir
+    value: $(Agent.BuildDirectory)\s
+  - name: LlvmBuildDir
+    value: $(LlvmSourceDir)\build
+  - name: LLVM_COMMIT
+    value: $[ dependencies.Configure.outputs['SetCommits.LLVM_COMMIT'] ]
+  - name: PKG_NAME
+    value: verona-llvm-install-x86_64-windows-$(BuildName)-$(LLVM_COMMIT)
+
+  steps:
+  - checkout: none
+  - template: ./steps/git-shallow-clone.yml
+    parameters:
+      RepoUri: $(LlvmRepoGitUri)
+      TargetDir: $(LlvmSourceDir)
+      CommitSha: $(LLVM_COMMIT)
+
+  - powershell: |
```

Comment:
`Choco` and `PowerShell` were used for `CMake` and `ninja` because the image we had didn't have any tools installed, but the standard Windows VM in Azure does. If you look at `ci.yml`, we're using the same image `windows-2019` and you'll see that it's all there and you can use `script` directly.

Reply:
removed choco and ninja installation steps